### DISCUSSION TITLE
同一个弹框中有两个form表单，修改操作时setFieldsValue只能声明一次，要怎么能解决？

### DISCUSSION THREAD

**Discussion by Wangxxz** [vbenjs/vue-vben-admin#3980] [Q&A] [answered]
<BasicForm @register="registerProjectForm" />
<BasicForm @register="registerGroup" />

const [registerProjectForm, { getFieldsValue, validate, **setFieldsValue**, resetFields, updateSchema  }] = useForm();

const [registerGroup, { **setFieldsValue**, }] = useForm();  第二个form表单使用setFieldsValue会报错‘ERROR: The symbol "setFieldsValue" has already been declared’，请问要如何解决

**Comment by likui628** [comment, marked as answer]
```
<BasicForm @register="registerProjectForm" />
<BasicForm @register="registerGroup" />

const [registerProjectForm, { getFieldsValue, validate, setFieldsValue, resetFields, updateSchema }] = useForm();

const [registerGroup, { setFieldsValue:setFieldsValueGroup, }] = useForm(); 
```
